docs/content/en/docs/getting-started/production-environment/gs-mgmt-cluster.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Create production cluster"
+weight: 40
+---
+
+EKS Anywhere supports a vSphere provider for production grade EKS Anywhere deployments.
+EKS Anywhere allows you to provision and manage Amazon EKS on your own infrastructure.
+
+This document walks you through setting up EKS Anywhere in a way that:
+
+* Deploys a management cluster on your vSphere environment
+* Deploys one or more workload clusters from the management cluster
+* Keeps the management cluster in place so you can use it to later upgrade and delete workload clusters
+
+% alert title="Important" color="primary" %
+
+In the initial release, EKS Anywhere clusters functioned as both workload and management clusters.
+CAPI functions moved back and forth before between temporary bootstrap clusters running on your administrative machine and the combined cluster.
+
+Separating management features into a separate, persistant management cluster
+provides a cleaner model for managing the lifecycle of workload clusters (to create, upgrade, and delete clusters), while workload clusters run user applications.
+This approach also allows for segmentation of different cluster types.
+
+% /alert %
+
+## Prerequisite Checklist
+
+EKS Anywhere needs to be run on an administrative machine that has certain [machine
+requirements]({{< relref "../install" >}}).
+An EKS Anywhere deployment will also require the availability of certain
+[resources from your VMware vSphere deployment]({{< relref "/docs/reference/vsphere/vsphere-prereq/_index.md" >}}).
+
+## Steps
+
+<!-- this content needs to be indented so the numbers are automatically incremented -->
+1. Generate a management cluster config:
+   ```bash
+   CLUSTER_NAME=sigb
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME \
+      --provider vsphere > eksa-mgmt-cluster.yaml
+   ```
+
+1. Modify the management cluster config (`eksa-mgmt-cluster.yaml`):
+
+   Create at least three control plane nodes and three worker nodes for a production cluster, to provide high availability and rolling upgrades. The following example contains that number, plus three separate nodes for etcd:
```

Comment:
I thought our default was 3 etcd, 2 control plane, and 3 worker. Did that change?

Reply:
The text says 3 cp and 3 worker nodes. The example shows 2 cp, 2 worker nodes, and 3 etcd. Is there a reason for only two CP nodes? Do you think  3 etcd, 2 control plane, and 3 worker is best for the default?

Reply:
CP doesn't need to reach quorum so there isn't a requirement on an odd number. They just need to be HA.

I will leave the default decision up to someone else on the team but I _thought_ 3:2:3 were the defaults for vSphere

Reply:
I'll put 3:2:3 then.